src/ex4/e2e/cypress/e2e/add-todo.cy.js

```diff
@@ -0,0 +1,23 @@
+describe("Add Todo Action", () => {
+	beforeEach(() => {
+		cy.visit("http://localhost:2000");
```

Comment:
your root URL should be a const (in a consts file) so it can be shared between different tests